ansible/roles/nginx/templates/etsin_nginx.conf

```diff
@@ -96,10 +96,6 @@ http {
 {% endif %}
     }
 
-    upstream matomo {
-        keepalive 15;
-        server {{ matomo.host | default('matomo.csc.local') }}:{{ matomo.port | default(80) }} weight=3;
-    }
```

Comment:
The matomo related blocks should not be completely removed from the template, since they are necessary for other envs than demo? See example from https://github.com/CSCfi/metax-ops/blob/centos/ansible/roles/nginx/templates/metax_nginx.conf#L135 how blocks in templates can be included or excluded conditionally.

Reply:
Thanks for the feedback. I checked our Matomo test instance, and it appears we are now using Matomo in every environment, including demo (and local). Also from what I understand, this code was added with the reverse proxy implementation. So, we will need to make it work in the provision, since it should be functional for every environment.

Reply:
Reverse proxy was set up here: https://github.com/CSCfi/etsin-ops/pull/17/files